android/src/com/mapswithme/maps/editor/OsmAuthFragmentDelegate.java

```diff
@@ -33,10 +34,12 @@ public OsmAuthFragmentDelegate(Fragment fragment)
 
   public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
   {
-    view.findViewById(R.id.login_osm).setOnClickListener(this);
-    view.findViewById(R.id.login_facebook).setOnClickListener(this);
-    view.findViewById(R.id.login_google).setOnClickListener(this);
-    view.findViewById(R.id.register).setOnClickListener(this);
+    for (@IdRes int childId : new int[] {R.id.login_osm, R.id.login_facebook, R.id.login_google, R.id.register})
+    {
+      final View v = view.findViewById(childId);
+      if (v != null)
```

Comment:
Ага, там есть же OsmAuthFragment где только логин в осме. 

> On Apr 15, 2016, at 6:43 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/editor/OsmAuthFragmentDelegate.java <https://github.com/mapsme/omim/pull/2899#discussion_r59894805>:
> 
> > @@ -33,10 +34,12 @@ public OsmAuthFragmentDelegate(Fragment fragment)
> >  
> >    public void onViewCreated(View view, @Nullable Bundle savedInstanceState)
> >    {
> > -    view.findViewById(R.id.login_osm).setOnClickListener(this);
> > -    view.findViewById(R.id.login_facebook).setOnClickListener(this);
> > -    view.findViewById(R.id.login_google).setOnClickListener(this);
> > -    view.findViewById(R.id.register).setOnClickListener(this);
> > +    for (@IdRes int childId : new int[] {R.id.login_osm, R.id.login_facebook, R.id.login_google, R.id.register})
> > +    {
> > +      final View v = view.findViewById(childId);
> > +      if (v != null)
> А разве каких-то из кнопок может не быть?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2899/files/10233c1d12282abd8144801c2407586c7080cf08..1570de35c0c95dbe594532640ffc2567aec5942c#r59894805>